README.md

```diff
@@ -324,7 +324,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Preset Cloud](https://preset.io/) - A hosted Apache Superset service. Forever free for teams of up to 5 users, featuring unlimited dashboards and charts, a no-code chart builder, and a collaborative SQL editor.
   * [PromptLeo](https://promptleo.com/) - Prompt engineering platform for creators and developers. It offers a prompt engineering library, forms, and API. The free plan provides one prompt formation, one prompt API endpoint, and 30 generations per month.
   * [PromptLoop](https://www.promptloop.com/) - Use AI and large language models like GPT-3 with a simple spreadsheet formula to transform, comprehend, and analyze text in Google Sheets. The first 2,000 credits are free each month.
-  * [ProxyCrawl](https://proxycrawl.com/) — Crawl and scrape websites without proxies, infrastructure, or browsers. We solve captchas for you and prevent you from being blocked. The first 1000 calls are free of charge.
+  * [Crawlbase](https://crawlbase.com/) — Crawl and scrape websites without proxies, infrastructure, or browsers. We solve captchas for you and prevent you from being blocked. The first 1000 calls are free of charge.
```

Comment:
We only lost things once. And listed entries have to meet all the criteria. Please close the ones that don’t and put it one entry only

Reply:
@ripienaar ,  **crawlbase.com** is more like a **Free Trial** not a **Free tier**. The 1000 request is fixed , It does not reset monthly.   Extra 9000 will be provide on adding Credit card which is definitely a Business Strategy , 

**Same Pull Request** ::   #3206 #3207  


![Screenshot from 2023-12-28 21-53-27](https://github.com/ripienaar/free-for-dev/assets/97863234/3c6b5486-a602-4cc3-be73-b6570c1f0393)
Their **Documenation** also mention it as a **Free Trial**

![Screenshot from 2023-12-28 21-53-22](https://github.com/ripienaar/free-for-dev/assets/97863234/3e5360e1-7ca0-4410-9dc4-4e35d61ec21f)

Reply:
Ok. Then we can close all these

Reply:
Thank you @madhuryadutta

Reply:
you're welcome :+1: